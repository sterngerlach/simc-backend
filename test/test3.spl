
{
    int x = 2;
    int y = 3;

    sprint("2 ^ 2 ^ 3 = ");
    iprint(x ^ x ^ y);
    sprint("\n");

    sprint("2 ^ 0 = ");
    iprint(2 ^ 0);
    sprint("\n");

    sprint("0 ^ 0 = ");
    iprint(0 ^ 0);
    sprint("\n");
}

