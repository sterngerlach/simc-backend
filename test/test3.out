2 ^ 2 ^ 3 = 256
2 ^ 0 = 1
0 ^ 0 = 1
